src/upload.ts

```diff
@@ -21,17 +21,17 @@ export async function upload (args: string[], internal = false) {
       // The `{type}|` prefix is used to determine which kind of CID is needed
       if (filepath_[1] !== '|') throw new Error('Invalid path format')
       switch (filepath_[0]) {
-        case 'r':
-          // raw file type
-          break
-        case 'm':
-          type = multicodes.SHELTER_CONTRACT_MANIFEST
-          break
-        case 't':
-          type = multicodes.SHELTER_CONTRACT_TEXT
-          break
-        default:
-          throw new Error('Unknown file type: ' + filepath_[0])
+      case 'r':
+        // raw file type
+        break
+      case 'm':
+        type = multicodes.SHELTER_CONTRACT_MANIFEST
+        break
+      case 't':
+        type = multicodes.SHELTER_CONTRACT_TEXT
+        break
+      default:
+        throw new Error('Unknown file type: ' + filepath_[0])
```

Comment:
In fact, these changes should trigger a linter errors. Try this on `group-income`, it will result in a linter error. Can you make sure this results in a linter error here too?